GUI/Main.cs

```diff
@@ -206,6 +207,7 @@ public Main(string[] cmdlineArgs, KSPManager mgr, GUIUser user, bool showConsole
                 minimizedContextMenuStrip.Renderer = new FlatToolStripRenderer();
                 ModListContextMenuStrip.Renderer = new FlatToolStripRenderer();
                 ModListHeaderContextMenuStrip.Renderer = new FlatToolStripRenderer();
+                LabelsContextMenuStrip.Renderer = new FlatToolStripRenderer();
```

Comment:
I think we need the same for the new submenus. They look good with a dark theme, but have a different colour with the standard theme:
![submenus](https://user-images.githubusercontent.com/28812678/70639043-2e60d400-1c3a-11ea-9841-7455497ca19e.png)

Reply:
Good catch!